examples/usecases/multi-gpu-data-parallel-training.ipynb

```diff
@@ -0,0 +1,469 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    MPI_SIZE = int(os.getenv("OMPI_COMM_WORLD_SIZE"))
</pre><p> I get error when I type this in a notebook.. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/855/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>